File: 0896-monotonic-array/0896-monotonic-array.py
class Solution:
    def isMonotonic(self, nums: List[int]) -> bool:
        i=0 
        j=1
        c=0
        for i in range(len(nums)-1):
            if nums[i]>nums[i+1]:
                c=-1
                break
                
        l=0
        for i in range(len(nums)-1):
            if nums[i]<nums[i+1]:
                l=-1
                break
        if c==0 or l==0:
            return True
        return False

File: 0896-monotonic-array/0896-monotonic-array.java
class Solution {
    public boolean isMonotonic(int[] nums) {
        int i=0;
        int c=0;
        for(i=0; i<nums.length-1; i++){
            if(nums[i]>nums[i+1]){
                c--;
                break;
            }
        }
        int l=0;
        for(i=0; i<nums.length-1 ; i++){
            if(nums[i]<nums[i+1]){
                l--;
                break;
            }
        }
        if(c==0 || l==0){
            return true;
        }
        return false;
    }
}